lib/js_defer.js

```diff
@@ -496,14 +501,20 @@ deferJsNs.DeferJs.prototype.cloneScriptNode = function(opt_script_elem) {
  * @param {number} opt_pos Optional position for ordering.
  */
 deferJsNs.DeferJs.prototype.addUrl = function(url, script_elem, opt_pos) {
-  this.log('Add to queue url: ' + url);
+  if (opt_pos !== undefined){
+    this.log('Insert to queue url at the position ' + opt_pos + ': ' + url);
+  }
+  else {
+    this.log('Push to queue url: ' + url);
+  }
+
   var me = this; // capture closure.
   this.submitTask(function() {
     me.removeNotProcessedAttributeTillNode(script_elem);
 
     var script = me.cloneScriptNode(script_elem);
     script.setAttribute('type', 'text/javascript');
-    var asyncScript = script.getAttribute('async') === 'true' || script.getAttribute('async') === '';
+    var asyncScript = script.getAttribute('async') !== null;
```

Comment:
this meen that `async` can be anything, including `false`, `'false'`, `undefined`, `'undefined'`, `'no'` ...

Reply:
According to https://developer.mozilla.org/fr/docs/Web/API/Element/getAttribute#Notes : 

> À peu près tous les navigateurs (Firefox, Internet Explorer, Opera, Safari, Konqueror et iCab, sans que cette liste soit exhaustive) renvoient null lorsque l'attribut demandé n'existe pas sur l'élément spécifié. La spécification DOM indique cependant que la valeur de retour correcte dans ce cas est en fait lachaîne vide , et certaines implémentation DOM auront ce comportement. Par conséquent, vous devriez utiliser element.hasAttribute() pour vérifier l'existence d'un attribut avant d'appeler getAttribute() s'il est possible que l'attribut n'existe pas sur l'élément spécifié.

I will have a test with `hasAttribute()`